doc/site/docs/01-getting-started/03-shelf-migration.md

```diff
@@ -356,8 +333,21 @@ void main() async {
 }
 ```
 
-## Learn More
+## Additional Resources
 
 - [Shelf Documentation](https://pub.dev/packages/shelf)
 - [Relic on GitHub](https://github.com/serverpod/relic)
 - [Serverpod](https://serverpod.dev)
+
+## Summary
```

Comment:
It feels like this is just another rehash. We should repeat the same things again and again.